src/statworx_theme/utils.py

```diff
@@ -116,8 +117,10 @@ def get_stwx_cmaps(as_hex=True) -> Dict[str, Any]:
     cmap_names = [cmap for cmap in plt.colormaps() if cmap.startswith("stwx:")]
     cmaps = [plt.get_cmap(cmap) for cmap in cmap_names]
     if as_hex:
+        from matplotlib import colors
```

Comment:
Why is matplotlib imported within the function? Its a hard dependency anyway.

Reply:
Good catch, I was wondering why mpl.colors.to_hex was not working and imported colors to test it, but forgot to remove